x-pack/platform/plugins/shared/alerting/server/application/backfill/methods/schedule/schemas/schedule_backfill_params_schema.ts

```diff
@@ -7,7 +7,10 @@
 
 import { schema } from '@kbn/config-schema';
 import { validateBackfillSchedule } from '../../../../../../common';
-import { MAX_SCHEDULE_BACKFILL_BULK_SIZE } from '../../../../../../common/constants';
+import {
+  MAX_SCHEDULE_BACKFILL_BULK_SIZE,
+  backfillInitiator,
+} from '../../../../../../common/constants';
 
 export const scheduleBackfillParamSchema = schema.object(
```

Comment:
so the idea is that the API requests will default to the a user initiator but using the client function directly, you can specify initiator and initiator ID?